### PR TITLE
900mhz

### PR DESCRIPTION
**Pull request by natey416**

### CODE REVIEW

File: src/mavric/CMakeLists.txt [hunk 1]
```diff
@@ -181,11 +181,11 @@ add_executable(${PROJECT_NAME}_CAN_Arm_Train src/CAN_Arm_Train.cpp)
...
-target_link_libraries(${PROJECT_NAME}_CAN_Arm_Train ${catkin_LIBRARIES})
-target_link_libraries(${PROJECT_NAME}_CAN_Arm_Train ${CMAKE_SOURCE_DIR}/${PROJECT_NAME}/lib/jetsontx/libCTRE_Phoenix.a)
-target_link_libraries(${PROJECT_NAME}_CAN_Arm_Train ${CMAKE_SOURCE_DIR}/${PROJECT_NAME}/lib/jetsontx/libCTRE_PhoenixCCI.a)
-target_link_libraries(${PROJECT_NAME}_CAN_Arm_Train ${CMAKE_SOURCE_DIR}/${PROJECT_NAME}/lib/jetsontx/libCTRE_PhoenixPlatform_socketcan.a)
-target_link_libraries(${PROJECT_NAME}_CAN_Arm_Train ${CMAKE_SOURCE_DIR}/${PROJECT_NAME}/lib/jetsontx/libCTRE_PhoenixCanutils.a)
+#target_link_libraries(${PROJECT_NAME}_CAN_Arm_Train ${catkin_LIBRARIES})
+#target_link_libraries(${PROJECT_NAME}_CAN_Arm_Train ${CMAKE_SOURCE_DIR}/${PROJECT_NAME}/lib/jetsontx/libCTRE_Phoenix.a)
+#target_link_libraries(${PROJECT_NAME}_CAN_Arm_Train ${CMAKE_SOURCE_DIR}/${PROJECT_NAME}/lib/jetsontx/libCTRE_PhoenixCCI.a)
+#target_link_libraries(${PROJECT_NAME}_CAN_Arm_Train ${CMAKE_SOURCE_DIR}/${PROJECT_NAME}/lib/jetsontx/libCTRE_PhoenixPlatform_socketcan.a)
+#target_link_libraries(${PROJECT_NAME}_CAN_Arm_Train ${CMAKE_SOURCE_DIR}/${PROJECT_NAME}/lib/jetsontx/libCTRE_PhoenixCanutils.a)
```
Comment (xGabeC) [inline src/mavric/CMakeLists.txt:188]: Can we just remove these now?

File: src/mavric/src/900MHz_Interface.py [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+        drive = rospy.Publisher("Drive_Train", Drivetrain, queue_size=10)
+        steer = rospy.Publisher("Steer_Train", Steertrain, queue_size=10)
+        status = rospy.Publisher("LoRa_Status", String, queue_size=10)
+        clients = rospy.Subscriber("client_count", Int32, client_cb, queue_size=10)
+        rospy.init_node("LoRa_Radio")
+
+        dataRX = struct
+        link = txfer.SerialTransfer(port)   # setup serial transfer on port specified
+        link.open() # open link between rover and LoRa
+        time.sleep(1)
```
Comment (xGabeC) [inline src/mavric/src/900MHz_Interface.py:57]: is there a need to sleep here?

**Review by xGabeC — APPROVED** [review]
Minor comments, should be fine to merge if you dont want to change

File: src/mavric/CMakeLists.txt (hunk 1 above)
Comment (natey416) [reply src/mavric/CMakeLists.txt:188]: Yep, I'll trim the cmake next week

File: src/mavric/src/900MHz_Interface.py (hunk 2 above)
Comment (natey416) [reply src/mavric/src/900MHz_Interface.py:57]: A sleep is used to make sure the serial transfer library opens the link fully and ensures the handshake is okay

Comment (natey416) [reply src/mavric/src/900MHz_Interface.py:57]: The 1 second was randomly chosen, I bet we can get that down

**Pull request merged**